paper/paper.bib

```diff
@@ -80,7 +80,7 @@ @Manual{Rpackage
   author       = {{R Core Team}},
   organization = {R Foundation for Statistical Computing},
   address      = {Vienna, Austria},
-  year         = YEAR,
+  year         = {n.y.},
```

Comment:
```suggestion
  year         = {2018},
```
The `YEAR` was somehow copied from the R FAQ: https://cran.r-project.org/doc/FAQ/R-FAQ.html#Citing-R
It should be an actual year (number).

@danielskatz

Reply:
I cannot write on this branch directly, I added the fix to 46c2973574af8bde0b5d7c035c50c75d3588fa8f